web/src/engine/main/src/headless/languageProcessor.ts

```diff
@@ -336,6 +336,7 @@ export class LanguageProcessor extends EventEmitter<LanguageProcessorEventMap> {
 
   public shutdown() {
     this.lmEngine.shutdown();
+    this.removeAllListeners();
```

Comment:
The relocated listener no longer can be removed via that `disconnect` method from `PredictionContext`, but the `EventEmitter` method `removeAllListeners` can ensure that it's cleaned up at the most critical time - when shutting down instances of the engine during unit testing.  That'd be the primary use-case for `PredictionContext.disconnect`, too.